src/float.rs

```diff
@@ -2008,4 +2013,9 @@ mod tests {
             assert!((Float::to_radians(deg) - rad).abs() < 1e-5);
         }
     }
+
+    #[test]
+    fn to_degrees_rounding() {
+        assert_eq!(1_f32.to_degrees(), 57.2957795130823208767981548141051703);
```

Comment:
When we forward to `std`, this fails on Rust 1.8 since it lacks rust-lang/rust#47919.  Maybe just limit this test to no-std builds? (with a comment explaining this reason)